mne/viz/evoked.py

```diff
@@ -637,6 +651,11 @@ def plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
         See matplotlib documentation for more details.
     axes : instance of matplotlib Axes | None
         Axes to plot into. If None, axes will be created.
+    dark_background : bool
+        Plot on dark background (black) or not (white).
+        It will be set to False by default in v0.16
+
+        .. versionadded:: 0.15.0
```

Comment:
I would rather have a generic `background_color` argument (or whatever we call it elsewhere). Then to decide the font color you just use the `np.mean(matplotlib.colors.colorConverter.to_rgb(background_color)) > 0.5` to decide whether to use black/white font